qemu/tests/cfg/sev_dhcert_boot.cfg

```diff
@@ -0,0 +1,12 @@
+- sev_dhcert_boot:
+    type = sev_dhcert_boot
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    vm_secure_guest_type = sev
+    vm_sev_reduced_phys_bits = 1
+    vm_sev_cbitpos = 51
+    vm_sev_policy = 7
+    sev_keyword = "sev-es"
+    sev_enable_check = "journalctl|grep -i ${sev_keyword}"
```

Comment:
Follow zhenchao's comment in 3825, do we want to add -w option for a strict single word check?